pkg/specgen/container_validate.go

```diff
@@ -145,6 +146,15 @@ func (s *SpecGenerator) Validate() error {
 	//	return errors.New("unrecognized option for cgroups; supported are 'default', 'disabled', 'no-conmon'")
 	//}
 
+	//set ulimits if not rootless
+	if len(s.ContainerResourceConfig.Rlimits) < 1 && !rootless.IsRootless() {
```

Comment:
Is this correct? @rhatdan Don't we have separate logic for determining rlimits based on the maximum allowed?

Reply:
fixed